libs/client/use-cases/part/list/shared/ui/src/lib/part-card/part-card-properties/part-card-properties.component.component.stories.ts

```diff
@@ -0,0 +1,44 @@
+import { svgProviders } from '@ddd-architecture/client/shared/infrastructure/taiga/feature';
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { PartCardPropertiesComponent } from './part-card-properties.component';
+import { UiPartCardProperty } from '../part-card.interface';
+
+export default {
+  title: 'PartCardPropertiesComponent',
+  component: PartCardPropertiesComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [],
+      providers: [...svgProviders],
+    }),
+  ],
+} as Meta<PartCardPropertiesComponent>;
+
+const Template: Story<PartCardPropertiesComponent> = (
+  args: PartCardPropertiesComponent
+) => ({
+  props: args,
+});
+
+export const Initial = Template.bind({});
+Initial.args = {
+  partCardProperties: null,
+};
+
+const mock: UiPartCardProperty[] = [
+  {
```

Comment:
please add a second use case where there will be more elements so that scroll-x appears